ipmiMgrApp/Db/afcv3.substitutions

```diff
@@ -41,22 +41,22 @@ file "power_mngr.db"
-            { FMC1Volt3V3     , 3           , 2       , $(fruid)     , $(id)  }
-            { FMC2Volt12V     , 4           , 2       , $(fruid)     , $(id)  }
-            { FMC2VoltVADJ    , 5           , 2       , $(fruid)     , $(id)  }
-            { FMC2Volt3V3     , 6           , 2       , $(fruid)     , $(id)  }
-            { FMC1Curr12V     , 1           , 3       , $(fruid)     , $(id)  }
-            { FMC1CurrVADJ    , 2           , 3       , $(fruid)     , $(id)  }
-            { FMC1Curr3V3     , 3           , 3       , $(fruid)     , $(id)  }
-            { FMC2Curr12V     , 4           , 3       , $(fruid)     , $(id)  }
-            { FMC2CurrVADJ    , 5           , 3       , $(fruid)     , $(id)  }
-            { FMC2Curr3V3     , 6           , 3       , $(fruid)     , $(id)  }
-            { TempFPGA        , 1           , 1       , $(fruid)     , $(id)  }
-            { TempUC          , 2           , 1       , $(fruid)     , $(id)  }
-            { TempClkSwitch   , 3           , 1       , $(fruid)     , $(id)  }
-            { TempDCDC        , 4           , 1       , $(fruid)     , $(id)  }
-            { TempRAM         , 5           , 1       , $(fruid)     , $(id)  }
+    pattern { attr            , sensinst    , type    , fruid        , prefix   , aliasprefix }
+            { FMC1Volt12V     , 1           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC1VoltVADJ    , 2           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC1Volt3V3     , 3           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2Volt12V     , 4           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2VoltVADJ    , 5           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2Volt3V3     , 6           , 2       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC1Curr12V     , 1           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC1CurrVADJ    , 2           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC1Curr3V3     , 3           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2Curr12V     , 4           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2CurrVADJ    , 5           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { FMC2Curr3V3     , 6           , 3       , $(fruid)     , $(id)    , alias_$(id) }
+            { TempFPGA        , 1           , 1       , $(fruid)     , $(id)    , alias_$(id) }
+            { TempUC          , 2           , 1       , $(fruid)     , $(id)    , alias_$(id) }
+            { TempClkSwitch   , 3           , 1       , $(fruid)     , $(id)    , alias_$(id) }
+            { TempDCDC        , 4           , 1       , $(fruid)     , $(id)    , alias_$(id) }
+            { TempRAM         , 5           , 1       , $(fruid)     , $(id)    , alias_$(id) }
```

Comment:
Please, document in the commit message why `noalias=#` is not an option if we use the `include` strategy.